pkgs/modules/nodejs/default.nix

```
@@ -118,7 +121,7 @@ in
       language = "javascript";
       extensions = [ ".js" ".jsx" ".ts" ".tsx" ".json" ".html" ];
       start = {
-        args = [ "${prettier}/bin/prettier" "--stdin-filepath" "$file" ];
+        args = [ "${prettierScript}/bin/prettier" "--stdin-filepath" "$file" ];
```

let's just change this to this
```suggestion
        args = [ "prettier" "--stdin-filepath" "$file" ];
```